core/src/dird/stats.cc

```diff
@@ -322,7 +330,7 @@ int StartStatisticsThread(void)
 
   statistics_initialized = true;
 
-  return 0;
+  return 1;
```

Comment:
Why do we have a return code here at all? All calls to "StartStatisticsThread()" that i saw never checked the return code. so can't we just change ther return code to void?

And can't we do the same with StartStatisticsThread() in sd_stats.cc also?

Reply:
In case the return code is required, we should check it in the code that calls StartStatisticsThread()

Reply:
You are right, the return value is not being used at all, but I believe we should make use of it. I already started using in the test, and I think we should even use the return value to send a message in case of a failed starting attempt. 
Let me know what you think of this

Reply:
Ok, then the return code of course needs to be checked and handeled everywhere. So the question is what to do if the start fails? Do we only log that or do we even stop the daemon in that case?